lib/getteams.js

```diff
@@ -0,0 +1,89 @@
+var ReturnTeamsActiveAtLHQ = function(hq,sector,cb) {
+
+	var now = moment();
+	var end = moment();
+	end.subtract(1, 'y');
+
+	UnitsToSearch = []
+
+	if (hq.EntityTypeId != 1) {
+		console.log("HQ Entity is not a unit, will resolve group")
+		$.ajax({
```

Comment:
yeap. given the swapping and reuse of these API calls i left them all as .ajax so its easier to compare